runtime/shared_common/shrinit.cpp

```diff
@@ -3952,6 +3953,30 @@ j9shr_getCacheSizeBytes(J9JavaVM *vm)
 	return ret;
 }
 
+/**
+ * Determine the type of shared class cache that is enabled. Either the current default, 
+ * Bootstrap Classes Only, or user defined shared cache, enabled via "-Xshareclasses" on the command line. 
+ * 
+ * @param [in] vm Pointer to the VM structure for the JVM
+ *
+ * @return J9SharedClassCacheMode enum that idicates the Shared Cache Class that is in effect
```

Comment:
"Shared Cache Class" should be "Shared Class Cache".

Reply:
Don't forget to update your commit message. It still says

`Add query to determine is Shared Class is the default SCC, returns TRUE is SCC is the default and FALSE otherwise`